00_Core/Scripts/Source/MilkQUEST.psc

```diff
@@ -571,16 +573,19 @@ Function MilkCycle(Actor akActor, int t)
 	SLA.UpdateActorExposure(akActor, t)
 
 	If BreastScaleLimit 
-		if MilkTick > ((MaidLevel+2)*2) && PiercingCheck(akActor) != 2
-			MilkCnt = ((MaidLevel+2)*2) as int
+		if MilkTick > MilkMax && PiercingCheck(akActor) != 2
+			MilkCnt = MilkMax as int
 			AddMilkFx(akActor, 1)
 			AddLeak(akActor)
 		else 
 			MilkCnt = MilkTick
 		endif
 	else
-		if MilkTick > ((MaidLevel+2)*2) && PiercingCheck(akActor) != 2
-			MilkTick = MilkTick - 1 * MilkTick / ((MaidLevel+2)*2)
+		if MilkTick > MilkMax && PiercingCheck(akActor) != 2
+			MilkTick = MilkTick - 1 * MilkTick / MilkMax
+			; suspicious calculation, please review
+			;  variant a) MilkTick - MilkTick / MilkMax
```

Comment:
Let me think about this a bit more. 'setMilkCurrent()' and related functions also need to be adjusted to support 'no Limit' properly.